prime-router/metadata/test_fhir_mapping/config.properties

```diff
@@ -0,0 +1,5 @@
+base.path.resource=./metadata/test_fhir_mapping
+supported.hl7.messages=ORU_R01,ORM_O01
+default.zoneid=+00:00
+additional.conceptmap.file=
+additional.resources.location=./metadata/fhir_mapping
```

Comment:
Test configuration references the original location for additional resources, so none need to be declared here